firefox-ios/firefox-ios-tests/Tests/XCUITests/ScreenGraphTest.swift

```diff
@@ -11,6 +11,25 @@ class ScreenGraphTest: XCTestCase {
     var navigator: MMNavigator<TestUserState>!
     var app: XCUIApplication!
 
+    func mozWaitForElementToNotExist(_ element: XCUIElement, timeout: TimeInterval? = TIMEOUT) {
```

Comment:
Since `ScreenGraphTest` inherits `XCTestCase` instead of `BaseTestCase`, I have to define `mozWaitForElementToNotExist()` and `waitUntilPageLoad()` here.